scanner/Makefile

```diff
@@ -0,0 +1,150 @@
+# Store tooling in a location that does not affect the system.
+GOBIN := $(CURDIR)/.gobin
+export GOBIN
+PATH := $(GOBIN):"$(PATH)"
+export PATH
+
+# Set to empty string to echo some command lines which are hidden by default.
+SILENT ?= @
+
+SHELL = /bin/bash
+
+TAG := # make sure tag is never injectable as an env var
+
+ifdef CI
+ifneq ($(NIGHTLY_TAG),)
+	TAG := $(NIGHTLY_TAG)
+else ifneq ($(RELEASE_TAG),)
+	TAG := $(RELEASE_TAG)
+endif
+endif
+
+ifeq ($(TAG),)
+	TAG = $(shell $(MAKE) -C ../ tag)
+endif
+
+UNAME_S := $(shell uname -s)
+UNAME_M := $(shell uname -m)
+
+GOOS := linux
+HOST_OS := linux
+ifeq ($(UNAME_S),Darwin)
+	HOST_OS := darwin
+endif
+
+ARCH := x86_64
+GOARCH := amd64
+ifeq ($(UNAME_M),arm64)
+	ARCH := aarch64
+	GOARCH := arm64
+endif
+
+GO_VERSION := $(firstword $(subst go version ,,$(shell go version)))
+EXPECTED_GO_VERSION := $(shell cat ../EXPECTED_GO_VERSION)
+ifneq ($(GO_VERSION),$(EXPECTED_GO_VERSION))
+$(error 'Go version "$(GO_VERSION)" does not match the expected "$(EXPECTED_GO_VERSION)" version.')
```

Comment:
note: this was required, as the normal tab indentation made it look like a command, which was not allowed:

```
Makefile:45: *** commands commence before first target.  Stop.
```

You can repro this by changing the version set in `EXPECTED_GO_VERSION`

Reply:
That's OK. The typical style I've seen elsewhere is indenting only the `ifeq`, `ifneq`, `ifdef` etc. Whatever we think is easier to read IMO.